sysutils/buildah/Makefile

```diff
@@ -0,0 +1,28 @@
+PORTNAME=	buildah
+DISTVERSION=	1.28.1.dev
+PORTVERSION=	1.28.1.dev
+CATEGORIES=	sysutils
+
+MAINTAINER=	dfr@FreeBSD.org
+COMMENT=	Manage Pods, Containers and Container Images
+WWW=		https://github.com/containers
+
+LICENSE=	GPLv2
+
+LIB_DEPENDS=	libgpgme.so:security/gpgme
+RUN_DEPENDS=	containers-common>=0:sysutils/containers-common \
+		containernetworking-plugins>=0:net/containernetworking-plugins \
+		ocijail:sysutils/ocijail
+BUILD_DEPENDS=	go-md2man:textproc/go-md2man \
+		bash:shells/bash \
+		go:lang/go \
```

Comment:
Replace this with `USES=go:no_targets`? This will make the port use whatever Go version is default in our Ports tree.